pom.xml

```
@@ -1,9 +1,13 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <properties>
+    <jenkins.version>2.120</jenkins.version>
+    <java.level>8</java.level>
+  </properties>
   <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.399</version><!-- which version of Jenkins is this plugin built against? -->
+    <version>3.2</version><!-- which version of Jenkins is this plugin built against? -->
```

```suggestion
    <version>4.12</version>
```

Comment is obsolete, you might also want to use a recent parent POM release